common/lib/common-utils/src/bufferBrowser.ts

```diff
@@ -100,17 +100,18 @@ export class IsoBuffer extends Uint8Array {
             return IsoBuffer.fromString(value, encodingOrOffset as string | undefined);
             // Capture any typed arrays, including Uint8Array (and thus - IsoBuffer!)
         } else if (value !== null && typeof value === "object" && isArrayBuffer(value.buffer)) {
-            // Support currently for full array, no view ports! (though it can be added in future)
-            assert(value.byteOffset === 0, 0x000 /* "nonzero isobuffer byte offset" */);
+            // Check that the view lies within the isobuffer
```

Comment:
I think these assertions are no longer needed, since the same checks are performed within the fromArrayBuffer function. If I remove the assertions, how does the shortCodeMap get updated?

Reply:
ok. I'll remove the assertions.

Reply:
Should I keep the entries in the shortCodeMap or also remove them there?

Reply:
You should leave them.  I know, it feels wrong, but the current sentiment is that there is only downside to attempting to remove the asserts.